From faf8f517018dc19d77e3280986d9bddb43670498 Mon Sep 17 00:00:00 2001
From: Arne Coucheron <arco68@gmail.com>
Date: Tue, 4 Feb 2020 02:50:34 +0100
Subject: [PATCH 2/2] sepolicy: Allow Snap to interact with gpuservice

avc: denied { call } for scontext=u:r:snap_app:s0:c512,c768 tcontext=u:r:gpuservice:s0 tclass=binder permissive=0 app=org.lineageos.snap

Change-Id: Ib9ae5076109fb0e0d5f9f3929b13e247d6df86e3
---
 common/private/snap_app.te | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/common/private/snap_app.te b/common/private/snap_app.te
index 969b3c4..03dbd2b 100644
--- a/common/private/snap_app.te
+++ b/common/private/snap_app.te
@@ -38,3 +38,6 @@ allow snap_app system_app_data_file:file { read write getattr };
 
 # Allow to read camera properties
 get_prop(snap_app, camera_prop)
+
+# Allow to interact with gpuservice
+binder_call(snap_app, gpuservice)
-- 
2.7.4

